content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -42,12 +40,12 @@ Accounting entries will depend on the accounting mode: *Continental* or *Anglo-S
 
 In *Anglo-Saxon* accounting, the costs of goods sold (COGS) are reported when products are sold or
 delivered. This means that the cost of a good is only recorded as an expense when a customer is
-invoiced for a product. *Interim Stock Accounts* are used for the input and output accounts, and are
-both *Asset Accounts* in the balance sheet.
+invoiced for a product. So, set the *Stock Input* and *Stock Output* accounts to a *Current Assets*
```

Comment:
I see on the UI that it is listed as "Stock Inut Account" and "Stock Output Account" under the "Stock Valuation" section of settings. I would add the section you need to go to for this, and make these terms GUI's. If they are re-written so the specific UI nae is listed, it can be a GUI, and might be more helpful to someone who is a little lost in accounting (like me).